packages/11ty/_includes/components/menu/list.js

```diff
@@ -13,19 +14,23 @@ module.exports = function(eleventyConfig) {
     const { navigation } = params
 
     const renderList = (items) => {
-      return items.map((item) => {
-        let listItem = ''
-        if (!item.children || item.children.length === 0) {
-          return `<li class="page-item">${menuItem(item)}</li>`
-        } else {
-          listItem += `<li class="section-item">${menuItem(item)}`
-          if (config.params.menuType !== 'brief') {
-            listItem += `<ul>${renderList(item.children)}</ul>`
-          }
-          listItem += '</li>'
-          return listItem
-        }
-      }).join('')
+      return html`
+        <ol>
```

Comment:
We might use the semantic `<menu>` element for this component, for reference https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu